tests/utils/daemonset/daemonset.go

```diff
@@ -73,3 +73,13 @@ func RedefineWithMultus(daemonSet *v1.DaemonSet, nadName string) *v1.DaemonSet {
 
 	return daemonSet
 }
+
+func RedefineWithImagePullPolicy(daemonSet *v1.DaemonSet, pullPolicy corev1.PullPolicy) *v1.DaemonSet {
+	if len(daemonSet.Spec.Template.Spec.Containers) > 0 {
```

Comment:
You don't need the "if" in this case. The "range" function will check that for you.